converters/fromZigbee.js

```diff
@@ -5998,6 +5998,29 @@ const converters = {
             }
         },
     },
+    xiaomi_curtain_acn003_motor_state: {
```

Comment:
Can you integrate this into the existing `aqara_opple` converter?

Reply:
sure. updated